internal/http/routes/legacy.go

```diff
@@ -80,12 +81,12 @@ func (r *LegacyAPIRoutes) Setup(g *gin.Engine) {
 	legacyGroup.POST("/api/logout", r.handle(r.legacyHandler.ApiLogout))
 
 	// router.GET(jp("/bookmark/:id/thumb"), withLogging(hdl.serveThumbnailImage))
-	legacyGroup.GET("/bookmark/:id/thumb", r.handle(r.legacyHandler.ServeThumbnailImage))
+	// legacyGroup.GET("/bookmark/:id/thumb", r.handle(r.legacyHandler.ServeThumbnailImage))
 	// router.GET(jp("/bookmark/:id/content"), withLogging(hdl.serveBookmarkContent))
-	legacyGroup.GET("/bookmark/:id/content", r.handle(r.legacyHandler.ServeBookmarkContent))
+	// legacyGroup.GET("/bookmark/:id/content", r.handle(r.legacyHandler.ServeBookmarkContent))
 	// router.GET(jp("/bookmark/:id/archive/*filepath"), withLogging(hdl.serveBookmarkArchive))
-	legacyGroup.GET("/bookmark/:id/archive/*filepath", r.handle(r.legacyHandler.ServeBookmarkArchive))
```

Comment:
as i mention before you are not remove `ServeBookmarkArchive` function like others but comment the legacy route.